src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2266,7 +2267,7 @@ because xbuild doesn't support framework reference assemblies.
 <Target Name="_GenerateJavaStubs"
   DependsOnTargets="_SetLatestTargetFrameworkVersion;_PrepareAssemblies;$(_AfterPrepareAssemblies);_PrepareNativeAssemblySources"
   Inputs="$(MSBuildAllProjects);@(_ResolvedUserMonoAndroidAssemblies);$(_AndroidManifestAbs);$(_AndroidBuildPropertiesCache);@(_AndroidResourceDest)"
-  Outputs="$(_AndroidStampDirectory)_GenerateJavaStubs.stamp">
+  Outputs="$(_AndroidStampDirectory)_GenerateJavaStubs.stamp;@(_TypeMapAssemblySource)">
```

Comment:
I still don't understand what is deleting these files:
```
Building target "_CompileNativeAssemblySources" completely.
Output file "obj/Release/android/typemap.jm.armeabi-v7a.o" does not exist.
...
[arm-linux-androideabi-as stderr] Error: can't open environment.armeabi-v7a.s for reading: No such file or directory
...
/Users/builder/jenkins/workspace/xamarin-android-pr-pipeline-release/xamarin-android/bin/Release/lib/xamarin.android/xbuild/Xamarin/Android/Xamarin.Android.Common.targets(2894,3): error XA3001: Could not compile native assembly file: environment.armeabi-v7a.s
```

I think this fix is fine for now, this target is just going to run a lot more than we want.

I will look into fixing up these targets later.